src/routing_table/prefix.rs

```diff
@@ -47,6 +47,11 @@ impl<T: Clone + Copy + Default + Binary + Xorable> Prefix<T> {
         self
     }
 
+    /// Returns `name` of the prefix
+    pub fn name(self) -> T {
```

Comment:
I don't think we want to expose this (the `Prefix` could eventually become a bitset or something and not have an underlying `name` of type `T` I guess).

I think the tentative agreement was to use `Prefix::lower_bound()` as the name to represent a group.